scripts/standard_configure_MINGW.sh

```diff
@@ -48,5 +48,5 @@ cmake ..
 -DKRATOS_GENERATE_PYTHON_STUBS=ON                                                                   \
 -DUSE_EIGEN_MKL=OFF
 
-# Buid
+# Build

```

Comment:
Line 10 can be changed to:
You can find a list `with` all the `compilation` options in INSTALL.md or here: